src/pages/settings/Profile/Contacts/ContactMethodDetailsPage.js

```diff
@@ -97,6 +97,16 @@ class ContactMethodDetailsPage extends Component {
         };
     }
 
+    componentDidUpdate(prevProps) {
+        const errorFields = lodashGet(this.props.loginList, [this.getContactMethod(), 'errorFields'], {});
+        const prevPendingFields = lodashGet(prevProps.loginList, [this.getContactMethod(), 'pendingFields'], {});
+
+        // Navigate to methods page on successful magic code verification
+        if (!errorFields.validateLogin && prevPendingFields.validateLogin === CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE) {
```

Comment:
@Pujan92 Could you add a comment to clarify why we are using the `validateLogin` value to check the success action?

Reply:
@mollfpr currently I have added a generic comment `Navigate to methods page on successful magic code verification`, do you think adding this `validateLogin property of errorFields & pendingFields is responsible to decide the status of the magic code verification` will be more helpful?